packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+  "app.components.NotFoundPage.back": "Tillbaka till startsidan",
+  "app.components.NotFoundPage.description": "Hittades inte",
+  "app.components.Official": "Officiell",
+  "app.components.Onboarding.label.completed": "% klar",
+  "app.components.Onboarding.title": "Kom igång videor",
+  "app.components.PluginCard.Button.label.download": "Ladda ner",
+  "app.components.PluginCard.Button.label.install": "Redan installerad",
+  "app.components.PluginCard.Button.label.support": "Stöd oss",
+  "app.components.PluginCard.compatible": "Kompatibel med din app",
+  "app.components.PluginCard.compatibleCommunity": "Kompatibel med samhället",
+  "app.components.PluginCard.more-details": "Mer detaljer",
+  "app.components.PluginCard.PopUpWarning.install.impossible.autoReload.needed": "AutoReload-funktionen måste vara inaktiverad. Starta din app med `yarn develop`.",
+  "app.components.PluginCard.PopUpWarning.install.impossible.environment": "Av säkerhetsskäl kan ett plugin bara laddas ner i en utvecklingsmiljö.",
+  "app.components.PluginCard.PopUpWarning.install.impossible.confirm": "Jag förstår!",
+  "app.components.PluginCard.PopUpWarning.install.impossible.title": "Att ladda ner är omöjligt",
+  "app.components.PluginCard.price.free": "Gratis",
+  "app.components.PluginCard.settings": "Inställningar",
+  "app.components.listPlugins.button": "Lägg till nytt plugin",
+  "app.components.listPlugins.title.none": "Inga plugins installerade",
+  "app.components.listPlugins.title.plural": "{number} plugins är installerade",
+  "app.components.listPlugins.title.singular": "{number} plugin är installerat",
+  "app.components.listPluginsPage.deletePlugin.error": "Ett fel inträffade vid avinstallation av pluginet",
+  "app.utils.SelectOption.defaultMessage": " ",
+  "app.utils.defaultMessage": " ",
+  "app.utils.placeholder.defaultMessage": " ",
+  "components.AutoReloadBlocker.description": "Kör Strapi med ett av de följande kommandon:",
+  "components.AutoReloadBlocker.header": "Reload funktionen krävs för detta plugin.",
+  "components.ErrorBoundary.title": "Någonting gick fel...",
+  "components.Input.error.attribute.key.taken": "Detta värde finns redan",
+  "components.Input.error.attribute.sameKeyAndName": "Kan inte vara lika",
+  "components.Input.error.attribute.taken": "Detta fältnamn finns redan",
+  "components.Input.error.contentTypeName.taken": "Detta namn finns redan",
+  "components.Input.error.custom-error": "{errorMessage} ",
+  "components.Input.error.validation.email": "Detta är inte en e-post",
+  "components.Input.error.validation.json": "Detta stämmer inte med JSON-formatet",
+  "components.Input.error.validation.max": "Värdet är för högt.",
+  "components.Input.error.validation.maxLength": "Värdet är för långt.",
+  "components.Input.error.validation.min": "Värdet är för lågt.",
+  "components.Input.error.validation.minLength": "Värdet är för kort.",
+  "components.Input.error.validation.minSupMax": "Kan inte vara överlägsen",
```

Comment:
? I'm guessing this is sort of related to another field, meaning, it should be "Kan inte vara högre än ..." with reference to another field value or label? The english one is also super weird.